packages/react-native-sdk/src/components/participants/ParticipantReaction.tsx

```diff
@@ -1,19 +1,28 @@
 import React, { useEffect, useState } from 'react';
-import { StreamReaction } from '@stream-io/video-client';
+import {
+  StreamVideoLocalParticipant,
+  StreamVideoParticipant,
+} from '@stream-io/video-client';
 import { StyleSheet, Text, View } from 'react-native';
 import { useCall } from '@stream-io/video-react-bindings';
-import { theme } from '../../../theme';
-import { StreamVideoRN } from '../../../utils';
+import { theme } from '../../theme';
+import { StreamVideoRN } from '../../utils';
 
-export type ReactionProps = {
-  reaction?: StreamReaction;
-  sessionId: string;
+export type ParticipantReactionProps = {
+  /**
+   * The participant whose info will be displayed.
+   */
+  participant: StreamVideoParticipant | StreamVideoLocalParticipant;
+  /**
+   * The duration after which the reaction should disappear.
+   */
```

Comment:
can we add the default value to the comment here pls

Reply:
Sure